app/helpers/otus/catalog_helper.rb

```diff
@@ -82,11 +90,22 @@ def otu_descendants_and_synonyms(
              otu_clones: Otu.where(name: otu.name, taxon_name_id: otu.taxon_name_id).where.not(id: otu.id).pluck(:id),
              similar_otus: s.inject({}){|hsh, n| hsh[n.id] = label_for_otu(n) ; hsh },
              nomenclatural_synonyms: ( (synonyms&.collect{|l| full_original_taxon_name_tag(l) || full_taxon_name_tag(l) } || []) - [a]).uniq, # This is labels, combinations can duplicate
+             # TODO: ignores project_scope_tag_id (will require a GA instead).
              common_names: (common_names ? otu_inventory_common_names(otu, language_alpha2) : []),
              descendants: []}
 
     if otu.taxon_name
-      descendants = otu.taxon_name.descendants.where(parent: otu.taxon_name).that_is_valid
+      descendants = otu.taxon_name.descendants
+        .where(parent: otu.taxon_name).that_is_valid
+
+      if project_scope_tag_id
```

Comment:
@kleintom Internal to catalog, rather than a tag Id. I think we should try to use an Otu scope as the join, that would let us pre-define any set of OTUs.  The concern would be performance, but let's experiment with it.

External to Catalog their might be a configure wrapper, that handles creating the OTU scopes.

So in the tag case the `otu_bounding_scope= Otu.joins(:tags).where(tags: {keyword_id: 123})`